code/modules/antagonists/heretic/knowledge/ash_lore.dm

```diff
@@ -223,7 +223,8 @@
 		When completed, you become a harbinger of flames, gaining two abilites. \
 		Cascade, which causes a massive, growing ring of fire around you, \
 		and Oath of Flame, causing you to passively create a ring of flames as you walk. \
-		You will also become immune to flames, space, and similar environmental hazards."
+		You will also become immune to flames, space, and similar environmental hazards. \
+		Also improves your ability to spread rust."
```

Comment:
I don't like putting "Also improves your ability to spread rust." in all of the other ascensions - too wordy and no one really cares if you're not already a rust heretic